src/apps/invitations/test_invite.py

```diff
@@ -46,7 +50,20 @@ async def test_invitation_retrieve(self):
         assert response.status_code == 200
 
         assert response.json()["result"]["appletId"] == 1
-        assert response.json()["result"]["role"] == "manager"
+        assert response.json()["result"]["role"] == Role.MANAGER
+
+    async def test_private_invitation_retrieve(self):
+        await self.client.login(self.login_url, "lucy@gmail.com", "Test123")
+
+        response = await self.client.get(
+            self.private_invitation_detail.format(
+                key="51857e10-6c05-4fa8-a2c8-725b8c1a0aa7"
+            )
+        )
+        assert response.status_code == 200
+
+        assert response.json()["result"]["appletId"] == 3
+        assert response.json()["result"]["role"] == Role.RESPONDENT
 
     @transaction.rollback
     async def test_admin_invite_manager_success(self):
```

Comment:
I would recommend using parametrized tests
https://docs.pytest.org/en/latest/contents.html/reference.html#pytest-mark-parametrize

Reply:
I have seen it. Parameters block will be big, so skipped such solution. Moreover, tried to make it more clean